.env.run.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export DATABASE_URL=postgres://odoo-13:odoo-13@localhost/library_development?sslmode=disable
+export SESSION_SECRET=mysessionsecret1forfirstsessionfirsttime
+export GOOGLE_CLIENT_ID=957604366444-tfje2deu51759js5l1floojl42bpo4d2.apps.googleusercontent.com
+export GOOGLE_CLIENT_SECRET=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
```

Comment:
you need to remove this file. and change that google application secrets immediately this shouldn't be committed to public repositories.